lib/src/components/NxToast/__tests__/NxToastContainer.test.tsx

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+import React, { useState } from 'react';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+import { screen, render, fireEvent } from '@testing-library/react';
+
+import userEvent from '@testing-library/user-event';
+
+import {reject, propEq } from 'ramda';
+
+import useToggle from '../../../util/useToggle';
+
+import NxToastContainer from '../NxToastContainer';
+import { NxToastContainerProps } from '../types';
+import NxToast from '../NxToast';
+import { NxErrorAlert } from '../../NxAlert/NxAlert';
+
+describe('NxToastContainer', function() {
+  const simpleProps: NxToastContainerProps = {
+    children: []
+  };
+
+  const quickRender = rtlRender<NxToastContainerProps>(NxToastContainer, simpleProps),
+      renderEl = rtlRenderElement(NxToastContainer, simpleProps);
+
+  it('renders a <div> with a class of nx-toast__wrapper', function() {
+    const component = quickRender();
+    const toastWrapper = component.container.children[0];
+
+    expect(toastWrapper.nodeName).toEqual('DIV');
+    expect(toastWrapper).toHaveClass('nx-toast__wrapper');
+  });
+
+  it('contains a<div> with a class of nx-toast__container', function() {
+    const component = renderEl();
+    const toastContainer = component?.querySelector('.nx-toast__container');
+
+    expect(toastContainer).toBeInTheDocument();
+    expect(component).toContainElement(toastContainer as HTMLElement);
+  });
+
+  it('tests NxToast mounting inside NxToastContainer, and unmounting when close button is clicked',
```

Comment:
"NxToastContainer tests NxToast mounting inside NxToastContainer..."? 

The test name should describe the behavior of `NxToastContainer`, not of the test itself.  This test also seems confusingly complicated and like it tests multiple things, some of which (like removal on close) are outside of `NxToast` and `NxToastContainer` entirely and are actually implemented in the test itself.  The only part of this that is actually `NxToastContainer`'s purview is that it renders the toasts that you give it.  I might test that by rendering an `NxToastContainer` containing a handful of different toasts and then asserting by text and maybe role that they're all in the output.